### ISSUE TITLE
Could I start and stop spinner in js file?

### ISSUE THREAD

**Issue by thearabbit** [SachaG/meteor-spin#31]
I would like to start or stop `spinner in js file`.
Ex:
```js
Template.hello.events({
............
spinner.start();
Meteor.call(......, function(err,res){
    if(!err){
       spinner.stop();
    }
})
})
```
Please help me.

**Comment by gkrizek** [comment]
I completed something similar to this with `Sessions` and `FlowRouter`.

I created an `if` statement in my template helper:
```
Template.hello.helpers({
    isLoading: function(state){
    	var loading = Session.get('loading');
    	return loading == state; 
    }
});
```
_Notice the `Session.get`_

Then in my template, created the `if` statement there:
```
{{#if isLoading true}}
     {{> spinner}}
{{else}}
      <div>Other Content</div>
{{/if}}
```

Then whenever you want to show the spinner, just set the `loading` Session to `true`. When you want it to stop set `loading` to `false`.
```
Session.set('loading', true);
```


Hope this helps!

_**I haven't tried it, but I'm pretty sure you don't have to put an `else` in the Template. Only have `if` to turn spinner on and off_
